calculator.js

```diff
@@ -32,16 +32,29 @@ operationButtons.forEach((button) => {
 
 function calc() {
   b = parseInt(inputFieldEl.innerHTML);
-  switch (operation) {
-    case "sum":
-      inputFieldEl.innerHTML = a + b;
-      break;
-
-    default:
-      break;
+  if (operation === 'sum') {
+    inputFieldEl.innerHTML = a+b
+  } else if (operation ==='sub') {
+    inputFieldEl.innerHTML = a-b
+  } else if(operation === 'multi'){
+    inputFieldEl.innerHTML = a*b
+  } else if(operation === 'div'){
+    inputFieldEl.innerHTML = a/b
   }
 }
 
-calcButton.addEventListener("click", () => {
-  calc();
-});
+calcButton.addEventListener("click",calc);
+
+clearButton.addEventListener("click",(e) =>{
+  inputFieldEl.innerHTML = 0;
+})
+
+
+
+backButton.addEventListener("click", (e) =>{
+ if(inputFieldEl.innerHTML.length === 2) {
```

Comment:
Why 2 and not 1? I think the reason for this bug is "number" class in backspace element.